website/site/content/guide/user/_index.md

```diff
@@ -9,49 +9,103 @@ This guide is an overview of the basic features to get you started. Focalboard i
+
+## With NGINX
+
+After following the standard [Mattermost install steps](https://docs.mattermost.com/install/install-ubuntu-1804.html#configuring-nginx-as-a-proxy-for-mattermost-server), edit `/etc/nginx/sites-available/mattermost` and add this section to it:
+
+```
+   location ~ /plugins/focalboard/ws/* {
+       proxy_set_header Upgrade $http_upgrade;
+       proxy_set_header Connection "upgrade";
+       client_max_body_size 50M;
+       proxy_set_header Host $http_host;
+       proxy_set_header X-Real-IP $remote_addr;
+       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+       proxy_set_header X-Forwarded-Proto $scheme;
+       proxy_set_header X-Frame-Options SAMEORIGIN;
+       proxy_buffers 256 16k;
+       proxy_buffer_size 16k;
+       client_body_timeout 60;
+       send_timeout 300;
+       lingering_timeout 5;
+       proxy_connect_timeout 90;
+       proxy_send_timeout 300;
+       proxy_read_timeout 90s;
+       proxy_pass http://backend;
+   }
+```
+
+Restart NGINX with `sudo systemctl restart nginx`.
+
+### With Apache (unofficial)
+
+After following the [install guide for Apache and Mattermost](https://docs.mattermost.com/install/config-apache2.html#configuring-apache2-as-a-proxy-for-mattermost-server-unofficial), modify the web sockets section in `/etc/apache2/sites-available` as follows:
+
+```
+ # Set web sockets
+  RewriteEngine On
+  RewriteCond %{REQUEST_URI} /api/v[0-9]+/(users/)?websocket|/plugins/focalboard/ws/* [NC,OR]
+  RewriteCond %{HTTP:UPGRADE} ^WebSocket$ [NC,OR]
+  RewriteCond %{HTTP:CONNECTION} ^Upgrade$ [NC]
+  RewriteRule .* ws://127.0.0.1:8065%{REQUEST_URI} [P,QSA,L]
+  ```
+
+Restart Apache with `sudo systemctl restart apache2`
+
+In Mattermost, select the Focalboard icon in the channel header to access boards for that channel.
+
 ## Adding new Boards
 
-1. Click "+ Add Board" at the lower left of the sidebar to add a new board to Focalboard.
-2. Pick "Project Tasks" from the list of templates.
-3. This shows the first view of the new board, which is a table of all tasks
+1. Select **+ Add Board** in the bottom left of the sidebar to add a new board to Focalboard.
+2. Pick **Project Tasks** from the list of templates.
+3. This shows the first view of the new board, which is a table of all tasks.
+
 ![image](./all%20tasks.png)
-4. Click on "By Status" in the sidebar to see a board view
+
+4. Click on "By Status" in the sidebar to see a board view.
+
 ![image](./by%20status.png)
+
 5. Click on the properties menu, and turn on "Priority" and "Date created". This will add those properties to the card display.
+
 ![image](./by%20status%20properties.png)
-6. You can likewise change the "Group By", "Filter", and "Sort" settings of the view
-7. In general, all changes you make are saved immediately
+
+6. You can likewise change the **Group By**, **Filter**, and **Sort** settings of the view.
+7. In general, all changes you make are saved immediately.
```

Comment:
Suggest "In general, all changes you make are saved immediately, and are visible to all other users".